registration/forms.py

```diff
@@ -1,483 +0,0 @@
-
-    def update_user(self, application, commit=True):
-        """
-        Update the UserProfile for the given application's user.
-        Also detect changes in design fields, store them in instance, and notify us by e-mail.
-        """
-        if hasattr(application.user, 'profile'):
-            application.user.profile.full_name = self.cleaned_data['full_name']
-            application.user.profile.subscribe_to_updates = self.cleaned_data['subscribe_to_updates']
-            application.user.profile.accepted_privacy_policy = timezone.now()
-        else:
-            application.user.profile = UserProfile(
-                full_name=self.cleaned_data['full_name'],
-                user_id=application.user.pk,
-                subscribe_to_updates=self.cleaned_data['subscribe_to_updates'],
-                accepted_privacy_policy=timezone.now(),
-            )
-        if commit:
-            with transaction.atomic():
-                application.user.profile.save()
-                application.save()
-
-        if self.restart_fields_changed():
-            if not application.email_addresses_verified():
-                messages.add_message(self.request, messages.INFO,
-                                     "Thank you for submitting these changes - we will build your instance to "
-                                     "apply them once your email address is verified.")
-                return
-            messages.add_message(self.request, messages.INFO,
-                                 "Thank you for submitting these changes - we will rebuild your instance to "
-                                 "apply them, and email you to confirm once it is up to date.")
-
-            # Notify us
-            if settings.VARIABLES_NOTIFICATION_EMAIL:
-                subject = 'Update required at instance {name}'.format(
-                    name=application.subdomain,
-                )
-                template = get_template('registration/fields_changed_email.txt')
-                text = template.render(dict(
-                    application=application,
-                    changed_fields=self.changed_data,
-                ))
-                sender = settings.DEFAULT_FROM_EMAIL
-                dest = [settings.VARIABLES_NOTIFICATION_EMAIL]
-                send_mail(subject, text, sender, dest)
-
-    def fields_with_errors(self):
-        """
-        Returns a list of fields that did not pass validation, in a
-        human-readable format.
-        """
-        return [self[field].label for field in self.errors]
-
-    @property
-    def _other_users(self):
-        """
-        Return a queryset for all users that are not the current user, if any.
-        """
-        users = User.objects.all()
-        if self.instance and self.instance.user_id:
-            users = users.exclude(pk=self.instance.user_id)
-        return users
-
-
-class LoginForm(NgFormValidationMixin, AuthenticationForm, metaclass=NgDeclarativeFieldsMetaclass):
```

Comment:
We'll need to keep the registration form, not sure this is the best place though.
You can move it if it doesn't fit well after the cleanup.

Reply:
I have unremoved them, I have keep the things as it is, for now, didn't change the place.